### ISSUE TITLE
Can I hide an axis but  not the ortho lines?

### ISSUE THREAD

**Issue by NinjaInShade** [leeoniya/uPlot#939]
![image](https://github.com/leeoniya/uPlot/assets/63664402/81c659f4-43e2-49e5-89b8-1fc2111efff1)

I have something like this, where I have multiple charts stacked on top of each other to display analogue/digital data. My goal here is to visually show all of these as "one" chart even though underneath it's multiple uPlot instances.

As you can see in the screenshot, the X axis shows for all charts. I can hide it with `{ show: false }`, but then the ortho lines also go away. Few questions:

1) can I show the ortho lines even if the axis is `{ show: false }`
  1a) can I make it so the ortho lines don't overflow the chart container so to speak, as they overflow a little bit (which is fine if you actually have a defined, shown, axis, but I don't in this case)?
2) so I know padding is added so that the axis labels aren't cut off, but can I have 0 padding on the chart but let the axis labels overflow? This is because the charts should be seamless between each other but the axis should be able to overflow. I guess what I may be looking for is way to add padding "inside" the canvas as opposed to around it

In other words, I'm making something similar to https://leeoniya.github.io/uPlot/demos/y-shifted-series.html but with series not having the same ranges/data so it is dynamic. I have started to make it with multiple chart instances but there a few issues (the main above) that prevent me from making the charts look seamless like your demo.

**Comment by NinjaInShade** [comment]
Another related question: I am currently hiding the bottom digital charts' y axes by setting `values: [null, null]`. Is this the best way and opposite to my first comment, how would I go about hiding the horizontal ortho lines as they introduce a bit of padding between charts I don't want and the values are always 0 or 1 so it's clear to see anyway?

**Comment by NinjaInShade** [comment]
![image](https://github.com/leeoniya/uPlot/assets/63664402/55babe6d-375d-42ee-b127-54c5bb6a2c73)

Here's a screenshot of when I hide the axes. Here you can notice the X-axis ortho lines only show on the last chart because all the others have the X-axis hidden. Also in this screenshot, you can see the right-axis text being cut off due to no padding in between charts. This padding problem I have also extends to where the charts meet - I'd like to have some spacing in-between the series but "inside" the canvas, not outside the charts.

**Comment by leeoniya** [comment]
there's no way to hide the axis but show the lines.

i think a better solution here would be to use a single plot for this. it will take a bit of data poking, but should be pretty straightforward.

if you attach your data here i can probably put together a demo of this. it's not clear what you're putting on the right y axis in the main plot, though :thinking:

**Comment by leeoniya** [comment]
actually, maybe 2 plots here would work well.

the bottom one would hold all the discrete series and do the same logic as https://leeoniya.github.io/uPlot/demos/y-shifted-series.html. the range there seems to be just 0/1 for each series, so shifting should be easy.

then you can simply set the x axis `size: 0` and disable label and tick rendering.

**Comment by leeoniya** [comment]
this should get you close enough: https://leeoniya.github.io/uPlot/demos/timeseries-discrete.html

<img width="1467" alt="image" src="https://github.com/leeoniya/uPlot/assets/43234/e7b2b71d-0ff7-4e57-81d8-b60e58856ef6">

**Comment by SystemParadox** [comment]
This is very helpful thank you!
